techniques/failures/F113.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html><html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml"><head><title>Failure of Success Criterion 1.2.5 due to not using available pauses in dialogue to provide audio descriptions of important visual content</title><link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"/></head><body>
+   <h1>Failure of Success Criterion 1.2.5 due to not using available pauses in dialogue to provide audio descriptions of important visual content</h1>
+   <section class="meta"><p class="id">ID: F113</p><p class="technology">Technology: failures</p><p class="type">Type: Failure</p></section>
+   <section id="applicability"><h2>When to Use</h2>
+      <p> Applies to all synchronized media. </p>
+   </section>
+   <section id="description"><h2>Description</h2>
+      <p>This describes a failure condition for all techniques involving audio descriptions. Audio descriptions can either be provided as part of the soundtrack's original narration or be added to audio during pauses in existing dialogue. If important actions, characters, scene changes, and on-screen text are only conveyed visually, appropriate pauses in dialogue need to be used to provide this information as audio descriptions.</p>
+        <p class="note">Not all pauses are usable for audio descriptions. If the pauses in dialogue are too short (for instance, less than 2 seconds), or do not occur in proximity to the visual content that needs to be described, they may not be appropriate for audio descriptions.</p>
+        <p>In situations where important visual information is being conveyed at the same time as non-spoken audio (such as music and sound effects), the technique of "audio ducking" can be used. This involves dropping the overall sound level so that it is easier to distinguish the narration that is added during pauses in dialogue.</p>
+        <p>This technique can work well with background music and sounds, but audio ducking has the potential to mask important audio information. Often such audio information can convey much of the sense of what is visually happening. There may be some pauses in dialogue where non-spoken audio information is so important that the addition of audio descriptions is unnecessary or inappropriate.</p>
+
+</p>
+   </section>
+    
+   <section id="tests"><h2>Tests</h2>
+      <section class="procedure"><h3>Procedure</h3>
+         <p>For each occurrence of synchronized time-based media:</p>
+         <ol>
+            <li> Check that all important visual information has been conveyed in the narration.</li>
```

Comment:
i think we need to be careful with the wording here. The normative language says "important visual details that cannot be understood from the main soundtrack alone". I know "important" is not, itself, defined, but I think it's better to stick with "important" rather than introducing a concept that is inferred but never stated explicitly: 
```suggestion
            <li> Check that all important visual information  that cannot be understood from the main soundtrack alone has been conveyed in the narration.</li>
```

Reply:
What is "the narration"? I thought this first line was about what was in the main soundtrack audio?

Reply:
It could either be in the main soundtrack or in the added audio description (or both).

Reply:
So I guess I don't understand what you are doing here. I'll re-read.

Reply:
One of the challenges we've been given with the Audio Description SC is that obviously if someone has done their work well, an instructional or educational video can have good narration without the need for the kind of intervention envisioned in this SC (adding a second audio track, etc). However, there is little scope in the SC wording for how an author could incorporate Audio Description into the pre-existing soundtrack (either while it is still in production by modifying the script, or in what I'll call post-production by adding more narrative into the main soundtrack.
The SC wording does not do a good job of considering adjustments made to a video that do _not_ involve the 2 main techniques that currently exist (adding a second, independently playable audio track; adding a separate version of the video with audio descriptions).
The definition of audio description is "narration added to the soundtrack".  That is the sense in which "narration" is intended to be used.

Reply:
I think we actually need a third check which is something like 'there is another version of this video which passes 1 and 2"

Reply:
If you get into CAVs in all failure techniques you will need to write them all.

Reply:
> However, there is little scope in the SC wording for how an author could incorporate Audio Description into the pre-existing soundtrack (either while it is still in production by modifying the script, or in what I'll call post-production by adding more narrative into the main soundtrack.

What about the definition of audio description?
"narration added to the soundtrack to describe important visual _details that cannot be understood from the main soundtrack alone_"

Reply:
You're forgetting note 3
> Where all of the video information is already provided in existing audio, no additional audio description is necessary.

If the only way we can fix this is to revise the unfortunate original definition of audio description, we can go there. I was hoping that succeeded in somewhat plugging this hole by the raft of changes we've got queued up to take to the WG, including https://github.com/w3c/wcag/pull/4327

Reply:
```suggestion
            <li>Check that all important visual information that cannot be understood from the main soundtrack alone has been conveyed in audio descriptions.</li>
```